apps/nrm/views/does-pv-have-children.html

```diff
@@ -0,0 +1,6 @@
+{{<partials-page}}
+  {{$page-content}}
+    {{#renderField}}does-pv-have-children{{/renderField}}
+    {{#input-submit}}continue{{/input-submit}}
+  {{/page-content}}
+{{/partials-page}}
```

Comment:
does this view need to created, is it not auto generated in HOF?

Reply:
This field below is auto generated if we don't override by using this file

does-pv-have-children-yes-amount

it shows within the yes option and outside of the radio buttons if this html is not in place.
@sulthan-ahmed